paparazzi2/src/main/java/app/cash/paparazzi/ClipSpec.kt

```diff
@@ -0,0 +1,8 @@
+package app.cash.paparazzi
+
+data class ClipSpec(
+  val frameSpec: FrameSpec,
+  val start: Long,
+  val end: Long,
+  val fps: Int
```

Comment:
Should we choose something more discrete for this? Like start + frameDelay + frameCount? I think we should offer a factory method that uses fps, but I think the model shouldn’t require us to have tricky rules around rounding. (Like what happens if we do 1 second at 30 FPS? Does that make 31 frames?)